src/contexts/RefreshContext.tsx

```diff
@@ -1,34 +1,52 @@
-import React, { useState, useEffect } from 'react'
-import useTabVisibility from 'hooks/useTabVisibility'
+import React, { useState, useEffect, useRef } from 'react'
 
 const FAST_INTERVAL = 10000
 const SLOW_INTERVAL = 60000
 
 const RefreshContext = React.createContext({ slow: 0, fast: 0 })
 
+// Check if the tab is active in the user browser
+const useIsBrowserTabActive = () => {
+  const isBrowserTabActiveRef = useRef(true)
+
+  useEffect(() => {
+    const onVisibilityChange = () => {
+      isBrowserTabActiveRef.current = !document.hidden
+    }
+
+    window.addEventListener('visibilitychange', onVisibilityChange)
+
+    return () => {
+      window.removeEventListener('visibilitychange', onVisibilityChange)
+    }
+  }, [])
+
+  return isBrowserTabActiveRef
+}
+
```

Comment:
I think this should be its own hook. You can use this other places where you want to detect if the browser tab is hidden or not.

Reply:
Yes but we don't. And I don't see a single use case where we want to use it. If one day we need it elsewhere, we will move it.